src/Microsoft.IdentityModel.Clients.ActiveDirectory/AuthenticationContext.cs

```diff
@@ -145,7 +145,7 @@ public Guid CorrelationId
         /// </summary>
         /// <param name="resource">Identifier of the target resource that is the recipient of the requested token.</param>
         /// <param name="clientId">Identifier of the client requesting the token.</param>
-        /// <returns>It contains Access Token, Refresh Token and the Access Token's expiration time.</returns>
+        /// <returns>It contains Access Token, its expiration time, user information.</returns>
```

Comment:
That's a device code? doesn't it contain info about the device?

Reply:
good catch, will fix it